plugin.yml

```
@@ -20,6 +20,9 @@ configuration:
       # in https://github.com/buildkite/zstash/blob/main/internal/commands/restore.go
       enum: ["s3", "nsc"]
       default: "s3"
+    registry-slug:
```

Looking at this more, should this just be:

```suggestion
    registry:
```

---

Done 👍🏻